test/browser_tests/spec_suites/shared/shared_office.js

```diff
@@ -0,0 +1,22 @@
+var Office = require( '../../page_objects/page_office.js' );
+
+describe( 'The offices template', function() {
+  it( 'should properly load in a browser', function() {
+    var page = new Office();
+    page.get( 'ProjectCatalyst' );
```

Comment:
So the reason it is here instead of `beforeEach` is some tests will use `page.get( 'PlainWriting' )` and `page.get( 'OfficeOfFinancialEducation' )`. I had all three in `beforeEach` and that meant that every test was instantiating two more Office objects than it needed. I'm not sure if it would make sense to maybe have three `describe` blocks for each type of office page, and the tests associated with each. All three are needed because that's the only way I could get full coverage of the office page content.